x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlExecutionInfo.java

```diff
@@ -397,7 +398,14 @@ public static class Cluster implements ToXContentFragment, Writeable {
         public static final ParseField TOOK = new ParseField("took");
 
         private final String clusterAlias;
-        private final String indexExpression; // original index expression from the user for this cluster
+        /**
+         * This cluster's indices as specified in the query. They may contain aliases, patterns, etc.
+         */
+        private final String originalIndices;
+        /**
+         * This cluster's concrete/resolved indices.
+         */
+        private final String concreteIndices;
```

Comment:
nit: I wonder if this should be a `String[]` instead.  Probably it depends on how we'll use it in the future.